packages/plans-grid/src/plans-table/style.scss

```diff
@@ -234,17 +235,43 @@
 	}
 }
 
+// it's OK to use $focused-launch-accent-blue variable here because the full-width mode is only used there
```

Comment:
This approach seems a bit brittle and it may confuse other developers, since it makes the button look different only based on the `.full-width` class.

It's not an issue created by this PR (and therefore not a blocker), but more of a sign that the plans grid package needs to be refactored and tidied up before it becomes too messy to maintain.

Reply:
I've refactored the styling for the button states.